db/migrate/20240302090207_remove_redundant_provider_index.rb

```diff
@@ -0,0 +1,8 @@
+class  RemoveRedundantProviderIndex < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction! if System::Database.postgres?
+
+  def up
+    remove_index :accounts, column: :provider_account_id if index_exists?(:accounts, :provider_account_id)
```

Comment:
why `if index_exists?` ?

Is it not determined by the schema whether it existed or not?

Reply:
to be sure that we don't fail if db was manually modified or if migration failed partially once and is rerun. Making it idempotent should always be good IMO.

Reply:
OK. Makes sense